ddl/ddl_api.go

```diff
@@ -5597,6 +5600,9 @@ func (d *ddl) CreatePrimaryKey(ctx sessionctx.Context, ti ast.Ident, indexName m
 		return dbterror.ErrTooLongIdent.GenWithStackByArgs(mysql.PrimaryKeyName)
 	}
 
+	if indexName.L != "" && indexName.L != mysql.PrimaryKeyNameL {
```

Comment:
Why not use `!strings.EqualFold(indexName.L, mysql.PrimaryKeyName)` instead of having another constant that is just lower case?